### PR TITLE
OPSEXP-574: Update search dependencies

### PR DESCRIPTION
**Pull request by gavincornwell**

### CODE REVIEW

File: helm/alfresco-content-services/charts/alfresco-sync-service/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 apiVersion: v1
-appVersion: 3.3.3.1
```
Comment (killerboot) [inline helm/alfresco-content-services/charts/alfresco-sync-service/Chart.yaml:2]: Is this related to search?

File: helm/alfresco-content-services/charts/alfresco-sync-service/Chart.yaml [hunk 2]
```diff
@@ -9,4 +8,4 @@ keywords:
 maintainers:
 - name: Alfresco Apps Repo Team
 name: alfresco-sync-service
-version: 3.0.6
+version: 3.0.7
```
Comment (killerboot) [inline helm/alfresco-content-services/charts/alfresco-sync-service/Chart.yaml:11]: Is this related to search?

**Review by killerboot — COMMENTED** [review]
Is it also worth removing the removed property "solr.content.dir" from here: https://github.com/Alfresco/acs-deployment/blob/0ffb1bece79776e93f66b4b52956305c24e33e25/helm/alfresco-content-services/charts/alfresco-search/values.yaml#L42
This property was removed in 2.0, see https://docs.alfresco.com/search-enterprise/concepts/external-properties-solr.html. But it is still present in older versions.

File: helm/alfresco-content-services/charts/alfresco-sync-service/Chart.yaml (hunk 1 above)
Comment (gavincornwell) [reply helm/alfresco-content-services/charts/alfresco-sync-service/Chart.yaml:2]: Fair point, no it's not, I will revert from this PR.

File: helm/alfresco-content-services/charts/alfresco-sync-service/Chart.yaml (hunk 2 above)
Comment (gavincornwell) [reply helm/alfresco-content-services/charts/alfresco-sync-service/Chart.yaml:11]: Fair point, no it's not, I will revert from this PR.

**Comment by gavincornwell** [comment]
Thank you also for highlighting the "solr.content.dir" property, we will investigate how to conditionally include that for older versions.

**Review by killerboot — APPROVED** [review]
